addons/stock/models/stock_quant.py

```diff
@@ -337,7 +338,11 @@ def _get_merge_query(self):
                                 package_id,
                                 lot_id
                             FROM stock_quant
-                            GROUP BY product_id, company_id, location_id, lot_id, package_id, owner_id, in_date
+        """
+        if location_ids:
+            query += """    WHERE location_id IN (%s)
+            """ % (" ,".join(str(l) for l in location_ids))
```

Comment:
`
consider using the below code
```
"""    WHERE location_id IN %s
            """, (tuple(location_ids),)
````

Reply:
`tuple` looks nicer for multiple location_ids, although for just one it adds a trailing comma. We'd need two slightly different functions for single and multiple instead of just the original implementation above.